### ISSUE TITLE
Limit access to gedis

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldfoundation/tft-stellar#22]
when gedis is exposed, everything like package management is exposed

**Comment by zaibon** [comment]
I think this is a duplicate of https://github.com/threefoldtech/jumpscaleX_core/issues/326

**Comment by robvanmieghem** [comment]
Sure is but I want to have it mentioned here to state that the functionality must be integrated in these services, it may certainly not be forgotten.

**Comment by robvanmieghem** [comment]
blocked on https://github.com/threefoldtech/home/issues/492

**Comment by zaibon** [comment]
ha yeah sorry didn't notice we were in the tf-stellar repo. my bad :-/

**Comment by robvanmieghem** [comment]
is blocked by the front proxy now, no acccess to gedis is needed directly
